develop/develop-images/build_enhancements.md

```diff
@@ -151,3 +151,46 @@ $ docker build --no-cache --progress=plain --secret id=mysecret,src=mysecret.txt
 #9     duration: 1.470401133s
 ...
 ```
+
+## Using SSH to access private data in builds
+
+> **Acknowledgment**:
+> Special thanks to [Tonis Tiigi](https://medium.com/@tonistiigi) for granting
```

Comment:
No need for this. You can just link to the post with "More info" style link.

Reply:
@andrewhsu does that work for you, or do you want me to keep this acknowledgement? CC: @L-Hudson

Reply:
if you can just point to tonis's blog post for more info, i think that would be suitable. doesn't have to specifically be worded as "Special thanks to..."